### ISSUE TITLE
Android get stuck on startup and restarting doesn't show biometrics login (samsung, xiaomi)

### ISSUE THREAD

**Issue by nop33** [alephium/alephium-frontend#352]
Multiple reports on this, all from Samsung or Xiaomi users.

<img src="https://github.com/alephium/alephium-frontend/assets/1579899/af5a1391-7177-4a89-9251-ed8d59fa4ce0" width="300" />

**Comment by mvaivre** [comment]
More info: apparently, a device restart solved the issue 🤔

**Comment by nop33** [comment]
Not always. We got an email:

> ** Cannot open wallet app on Samsung S20**
>
> Hey guys, i just got a new to me S20fe and i installed the app from the Play Store but it won't open. I get the welcome screen but it doesn't get past there and ask for a seed phrase or make a new account. My old phone (s10) is useable and i can access the wallet from there. I have uninstalled and reinstalled tge app. Any ideas?

**Comment by nop33** [comment]
This indicates that there is a problem with accessing the secure storage on some devices...

**Comment by nop33** [comment]
We have the same issue with Xiaomi phones

**Comment by nop33** [comment]
This might be related: https://stackoverflow.com/a/67195378/1644591

Let's disable this flag https://docs.expo.dev/versions/latest/config/app/#allowbackup

**Comment by nop33** [comment]
@mvaivre did you have a chance in reproducing this on the Samsung phone?

**Comment by mvaivre** [comment]
No, I didn't get to reproduce it. Will try again today with the latest version to make sure.

**Comment by mvaivre** [comment]
Can't reproduce on my Samsung phone.

**Comment by mvaivre** [comment]
Didn't get any new reports. Closing, but feel free to reopen if this happens on your device.
